iOS/Projects/Features/Record/Sources/WorkoutSelectScene/View/WorkoutSelectViewController.swift

```diff
@@ -28,9 +40,10 @@ final class WorkoutSelectViewController: UIViewController {
     return label
   }()
 
-  lazy var workoutCardCollectionView: UICollectionView = {
+  lazy var workoutTypesCollectionView: UICollectionView = {
     let collectionView = UICollectionView(frame: .zero, collectionViewLayout: makeCollectionViewLayout())
-    collectionView.register(WorkoutCardCell.self, forCellWithReuseIdentifier: WorkoutCardCell.identifier)
+    collectionView.register(WorkoutSelectTypeCell.self, forCellWithReuseIdentifier: WorkoutSelectTypeCell.identifier)
+    collectionView.backgroundColor = UIColor.clear
```

Comment:
p3: 간단하게 줄일 수 있을 것 같습니다.
```suggestion
    collectionView.backgroundColor = .clear
```

Reply:
감사합니다!